src/RRule.php

```diff
@@ -316,6 +323,8 @@ public function __call(string $methodName, array $args)
             );
         }
 
-        return call_user_func_array([$this->rrule, $methodName], $args);
+        return call_user_func_array(function (...$args) use ($methodName) {
+            return $this->rrule->$methodName(...$args);
+        }, $args);
```

Comment:
This looks pointless. What has been fixed here?

Reply:
```php
Line   RRule.php                                                        
 ------ ----------------------------------------------------------------- 
  326    Parameter #1 $callback of function call_user_func_array expects  
         callable(): mixed, array{Recurr\Rule, string} given.    
```

Reply:
https://github.com/phpstan/phpstan/issues/1242#issuecomment-439716243

Reply:
https://github.com/phpstan/phpstan/issues/1210#issuecomment-404482691

Reply:
Oi. Then ignore the error. This abomination isn't going to be merged.